profiles/coreos/arm64/package.accept_keywords

```diff
@@ -2,6 +2,8 @@
 # Keep these in alphabetical order.
 
 =app-admin/sudo-1.8.20_p2 ~arm64
+=app-arch/bzip2-1.0.6-r8 ~arm64
+=app-arch/unrar-5.5.8 ~arm64
```

Comment:
I'd say leave `unrar` line out.  If the package is removed this would need to be cleaned up, which seems would be easy to forget.